src/runtime/hexagon_remote/sim_host.cpp

```diff
@@ -84,6 +84,15 @@ int init_sim() {
                 printf("HexagonWrapper::SetTracing failed: %d\n", status);
                 return -1;
             }
+        } else {
+            const char *T = getenv("HL_HEXAGON_SIM_MEM_TRACE");
+            if (T && T[0] != 0) {
```

Comment:
nit: this just checks that the var is defined and non-empty, so defining HL_HEXAGON_SIM_MEM_TRACE=0 would actually enable tracing? that seems... odd.